src/server/lib/idapi/subscriptions.ts

```diff
@@ -35,6 +40,19 @@ export const parseSubscriptionData = (data: string): SubscriptionData => {
 	};
 };
 
+export const parseUnsubscribeAllData = (data: string): UnsubscribeAllData => {
+	const [userId, timestamp] = data.split(':');
+
+	if (!userId || !timestamp) {
+		throw new Error('Invalid unsubscribe-all data');
+	}
+
+	return {
+		userId,
+		timestamp: +timestamp,
```

Comment:
The `+` is very elegant! I would have probably done `Number(timestamp)` because it feels more explicit to me that it's making a number, but that's very much a personal preference.

Reply:
Yeah it's a good question. I went for consistency with the existing method just above this which also uses `+timestamp`. What do you reckon? (I think we should either update both or update neither).